paper-input-error.html

```diff
@@ -65,20 +65,5 @@
     behaviors: [
       Polymer.PaperInputAddonBehavior
     ],
-
-    properties: {
-      /**
-       * True if the error is showing.
-       */
-      invalid: {
```

Comment:
I don't think this can be removed, since it would constitute a breaking change, no? What if someone is using this attribute?